CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.6.3
```

Comment:
i think this might be considered a minor version change rather than a patch, but i don tkonw the pedantics of that

Reply:
@losvedir what do you say

Reply:
I agree with @sghiaa. Pre 1.0 it doesn't matter so much according to [the spec](https://semver.org):

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

so the usual idea of patch vs minor changes and what's breaking and what's not doesn't really apply.

What we've been doing so far is roughly: bug fixes and tweaks that aren't really intended to change the behavior of the app count as patch updates, while new or changing features qualify as minor updates. Since this would change the page a fair bit, I think bumping to `0.7.0` makes sense.